modulemd/tests/test-modulemd-moduleindex.c

```diff
@@ -1002,6 +1002,22 @@ module_index_test_get_default_streams (void)
 }
 
 
+static void
+module_index_test_dump_empty_index (void)
+{
+  g_autoptr (GError) error = NULL;
+  g_autofree gchar *yaml = NULL;
+  g_autoptr (ModulemdModuleIndex) index = modulemd_module_index_new ();
+
+  g_assert_nonnull (index);
+  g_assert_true (MODULEMD_IS_MODULE_INDEX);
```

Comment:
The compiler is complaining about this assertion:
```
../modulemd/tests/test-modulemd-moduleindex.c: In function 'module_index_test_dump_empty_index':
../modulemd/tests/test-modulemd-moduleindex.c:1013:18: warning: the address of 'MODULEMD_IS_MODULE_INDEX' will always evaluate as 'true' [-Waddress]
 1013 |   g_assert_true (MODULEMD_IS_MODULE_INDEX);
      |                  ^~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/glib-2.0/glib/gmacros.h:900:25: note: in definition of macro 'G_LIKELY'
  900 | #define G_LIKELY(expr) (expr)
      |                         ^~~~
../modulemd/tests/test-modulemd-moduleindex.c:1013:3: note: in expansion of macro 'g_assert_true'
 1013 |   g_assert_true (MODULEMD_IS_MODULE_INDEX);
      |   ^~~~~~~~~~~~~
```

Reply:
Shouldn't it be `g_assert_true (MODULEMD_IS_MODULE_INDEX (index));`?

Reply:
Yes, it should be. Weird that it wasn't a compilation error though.